packages/ui/src/elements/ReactSelect/index.scss

```diff
@@ -6,6 +6,7 @@
   }
 
   .react-select {
+    z-index: 3;
```

Comment:
would a `:focus-within` selector help here?

I'd test with a lexical toolbar + lexical block with this select field in it

Reply:
works well!